bin/src/prove.rs

```diff
@@ -0,0 +1,74 @@
+use prover::{ChunkProof, WitnessBlock};
+use std::panic::{catch_unwind, AssertUnwindSafe};
+
+#[cfg(feature = "batch-prove")]
+pub fn prove_batch(id: &str, chunk_proofs: Vec<ChunkProof>) {
+    let chunk_hashes_proofs = chunk_proofs
```

Comment:
panic if chunk_proofs.is_empty()?   in case any unexpected err

Reply:
<img width="654" alt="image" src="https://github.com/scroll-tech/scroll-prover/assets/2833376/96cd1236-9d53-4cc7-81f1-69d84c15e7a0">

oh i guess it will panic here?

Reply:
and   let real_chunk_count = chunk_hashes_proofs.len();
        assert!((1..=MAX_AGG_SNARKS).contains(&real_chunk_count));

oh no problems.